envbuilder.go

```diff
@@ -334,6 +337,7 @@ func run(ctx context.Context, opts options.Options, execArgs *execArgsInfo) erro
 		} else {
 			// If a Dockerfile was specified, we use that.
 			dockerfilePath := filepath.Join(buildTimeWorkspaceFolder, opts.DockerfilePath)
+			opts.Logger(log.LevelInfo, "Using Dockerfile: %s", opts.DockerfilePath)
```

Comment:
How about something like "Building in Dockerfile-only mode"?

Reply:
I'll update the corresponding line for "Devcontainer mode" also 👍